src/NUnitFramework/framework/Constraints/ConstraintExpression.cs

```diff
@@ -8,10 +8,10 @@
 // distribute, sublicense, and/or sell copies of the Software, and to
 // permit persons to whom the Software is furnished to do so, subject to
 // the following conditions:
-// 
+//
```

Comment:
The weird part is that Visual Studio won't show differing whitespace. Only GitHub and the git command line. Wish this could be set for GitHub via .gitattributes or something.